src/components/DateTimePicker/Internal/Tests/util/commonUtil.tsx

```diff
@@ -177,29 +186,38 @@ export function selectCell(text: string | number, index = 0) {
 
 export function clickButton(type: string) {
   let matchBtn: HTMLButtonElement;
-  Array.from(document.querySelectorAll('button')).forEach((btn) => {
-    if (btn.className.includes(`-header-${type}-btn`)) {
-      matchBtn = btn;
-    }
-  });
 
-  fireEvent.click(matchBtn);
+  if (canUseDocElement()) {
+    Array.from(document.querySelectorAll('button')).forEach((btn) => {
```

Comment:
could something like this work and avoid the if wrapper? 

```suggestion
    Array.from(document?.querySelectorAll('button') || []).forEach((btn) => {
```